test/jdk/javax/swing/JOptionPane/OptionPaneInput.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
```

Comment:
Isn't the "Classpath" exception only applicable to source code files?

Reply:
Good catch, I've removed the classpath portion of the copyright. Can you check over the copyright again to make sure it's correct?